app/workers/delivery_request_worker.rb

```diff
@@ -10,7 +10,7 @@ class RateLimitExceededError < StandardError; end
       email_id = msg["args"].first
       queue = msg["args"].second
       GovukStatsd.increment("delivery_request_worker.rescheduled")
-      DeliveryRequestWorker.set(queue: queue).perform_in(30.seconds, email_id, queue)
+      DeliveryRequestWorker.set(queue: queue).perform_in(5.minutes, email_id, queue)
```

Comment:
I'm confused: why do we do this at all if Sidekiq is doing it for us?

Reply:
This code only runs after sidekiq is given up. I'm assuming the original intent was that if Sidekiq is going to give up and it's given up because of our RateLimiting then we should keep on trying.